_pages/develop/map.md

```diff
@@ -0,0 +1,682 @@
+---
+title: Developing with px-map
+moduleName: view-develop-map
+pathToRoot: ../../
+layout: default
+---
+
+<style>
+  .gif {
+    width: 100%;
+    margin-top: .25rem;
+  }
+</style>
+
+# Overview
+Px-map is a lightweight framework for building interactive maps with web
+components and Polymer. The framework includes a base mapping component (`<px-map>`)
+that can be used with many different subcomponents to solve common mapping UI
+problems, like plotting geographic data.
+
+The current major release of px-map uses the open source [Leaflet](http://leafletjs.com/)
+library to support displaying a map with base tiles and geospatial overlays.
+Future iterations of the component may offer the ability to use a different
+library in place of Leaflet, while keeping the same basic API.
+
+Out of the box, `px-map` includes the basic things needed to draw and plot data
+on an interactive map. It is also extensible — development teams are encouraged
+to build their own subcomponents that add features needed for their application.
+We currently do not support visualizations, but any open source Leaflet plugin
+can be easily converted to be a px-map custom layer.
+
+# Basic concepts
+
+Use the map to plot geographic data such as individual assets or clusters of
+assets and information about them. The data concepts underlying the map include:
+
+- **Base layer:** The base layer of a map renders tile images that include
+information such as streets, mountains, and national parks. For example,
+satellite, terrain, and map view are all different base layer styles.
+- **Overlay:** Information that is put on top of a map's base layer that is
+specific to the purpose of that map. For example, traffic data or bike lane
+information might be set as an overlay on a road map.
+- **Control:** A map layer that allows the user to manipulate the map, e.g. by
+zooming in, out, or to their current location.
+- **CRS:** A CRS is a coordinate reference system (also known as a coordinate
+relationship system or coordinate projection system) used to project
+geographical points into pixel coordinates.
+
+## Graphic: map layers
+
+# Subcomponents
+
+The following subcomponents are available for use with px-map:
+
+**Base map**
+
+- `<px-map>`: Draws the underlying map, sets location/zoom and notifies updates
+when the user interacts with the map, enables and disables interactions,
+orchestrates subcomponents
+
+**Tile layers**
+
+- `<px-map-tile-layer>`: Calls a tile service API to fetch underlying tile images
```

Comment:
Should we specify that the default is Open Street Map?

Reply:
There isn't a default tile layer as far as I know. We use Open Street Map for our demos, but if you don't specify a tile layer, the map will just be blank.

Reply:
👍